### PR TITLE
Updated Req.txt

### LINKED ISSUE #115: Invalid Syntax in savedb.py
Hi guys,

First of all, thank you for handling this project which seems really interesting.

When I try to test your tool, I have an error concerning savedb.py. Here is the traceback I have : 

> Traceback (most recent call last):
>   File "torBot.py", line 6, in <module>
>     from modules import (bcolors, getemails, pagereader, getweblinks, updater,
>   File "/usr/toolz/TorBot/modules/savedb.py", line 31
>     print "Error '{0}' occurred. Arguments {1}.".format(e.message, e.args):
>                                                ^
> SyntaxError: invalid syntax

Did the mistake came from my side and you can I solve that?

Thank you for your help!

EDIT :  Just saw that this error will be corrected in TorBot 1.3, am I right?

### PR DESCRIPTION
**Pull request by PSNAppz**
Fixes #115 

### Changes Proposed
- Updated requirements.txt 
- Fixed a format error in savedb.py

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -5,3 +5,5 @@ requests==2.18.4
 requests_mock==1.4.0
 tldextract==2.2.0
 yattag==1.10.0
+mysqlclient==1.3.13
```
Comment (KingAkeem) [inline requirements.txt]: Not sure if this is a pip install-able module yet. Didn't work for me personally.

Comment (PSNAppz) [reply requirements.txt]: https://stackoverflow.com/questions/35991403/pip-install-unroll-python-setup-py-egg-info-failed-with-error-code-1

Comment (KingAkeem) [reply requirements.txt]: I followed these steps and it didn't solve the issue.

Comment (PSNAppz) [reply requirements.txt]: Did you try other solutions?

Comment (KingAkeem) [reply requirements.txt]: I'll try the rest in a little and let you know if they work.

Comment (PSNAppz) [reply requirements.txt]: Ok

Comment (PSNAppz) [reply requirements.txt]: Can you post the error message?

Comment (shivankar-madaan) [reply requirements.txt]: Failing for me as well
here's the error code
`root@kali:~/github/TorBot# pip install mysqlclient
Collecting mysqlclient
  Using cached https://files.pythonhosted.org/packages/ec/fd/83329b9d3e14f7344d1cb31f128e6dbba70c5975c9e57896815dbb1988ad/mysqlclient-1.3.13.tar.gz
    Complete output from command python setup.py egg_info:
    /bin/sh: 1: mysql_config: not found
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-41snh5qi/mysqlclient/setup.py", line 18, in <module>
        metadata, options = get_config()
      File "/tmp/pip-install-41snh5qi/mysqlclient/setup_posix.py", line 53, in get_config
        libs = mysql_config("libs_r")
      File "/tmp/pip-install-41snh5qi/mysqlclient/setup_posix.py", line 28, in mysql_config
        raise EnvironmentError("%s not found" % (mysql_config.path,))
    OSError: mysql_config not found
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-41snh5qi/mysqlclient/
`

Comment (PSNAppz) [reply requirements.txt]: @shivankar-madaan Try this ` pip install --upgrade setuptools`

Comment (shivankar-madaan) [reply requirements.txt]: yes tried all the steps in the stackoverflow link

Comment (PSNAppz) [reply requirements.txt]: See this also
https://stackoverflow.com/questions/25865270/how-to-install-python-mysqldb-module-using-pip

**Pull request merged**